tests/unit/source_code/samples/s3fs-python-compatibility-catalog.yml

```diff
@@ -0,0 +1,29 @@
+%YAML 1.2
+---
+identifier:
+  name: leeds
+  # All versions are compatible
+top_level: leeds
+packages:
+  - name: leeds
+    compatibility: full
+    failures:
+      - some failure 1
+  - name: databricks.sdk.service.compute
+    compatibility: full
+    failures:
+      - some failure 1
+---
+identifier:
+  name: s3fs
+  # all versions are incompatible
+top_level: s3fs
```

Comment:
This is a test resource, how this will be read by prod?..

Reply:
I can add it to pre-known packages. First the test I just copied existing test config.